docs/docusaurus/docs/guides/connecting_to_your_data/fluent/cloud/how_to_connect_to_data_on_gcs_using_spark.md

```diff
@@ -0,0 +1,112 @@
+---
+title: How to connect to data on GCS using Spark
+tag: [how-to, connect to data]
+description: A technical guide demonstrating how to connect Great Expectations to dat stored on Google Cloud Server using Spark.
+keywords: [Great Expectations, Google Cloud Server, GCS, Spark]
+---
+
+<!-- Import statements start here. -->
+import Prerequisites from '/docs/components/_prerequisites.jsx'
+
+<!-- ### 1. Import GX and instantiate a Data Context -->
+import ImportGxAndInstantiateADataContext from '/docs/components/setup/data_context/_import_gx_and_instantiate_a_data_context.md'
+
+<!-- ### 3. Add GCS data to the Datasource as a Data Asset -->
+import BatchingRegexExplaination from '/docs/components/connect_to_data/cloud/_batching_regex_explaination.mdx'
+
+<!-- Next steps -->
+import AfterCreateNonSqlDatasource from '/docs/components/connect_to_data/next_steps/_after_create_non_sql_datasource.md'
+
+## Introduction
+
+In this guide we will demonstrate how to use Spark to connect to data stored on Google Cloud Server.  In our examples, we will specifically be connecting to `.csv` files.
+
+## Prerequisites
+
+<Prerequisites>
+
+- [An installation of GX set up to work with GCS](/docs/guides/setup/optional_dependencies/cloud/how_to_set_up_gx_to_work_with_data_on_gcs.md)
+- Access to data on a GCS bucket
+- A passion for data quality
+
+</Prerequisites> 
+
+## Steps
+
+### 1. Import GX and instantiate a Data Context
+
+<ImportGxAndInstantiateADataContext />
+
+### 2. Create a Datasource
+
+We can define a GCS datasource by providing three pieces of information:
+- `name`: In our example, we will name our Datasource `"MyGcsDatasource"`
```

Comment:
@Rachel-Reverie Same comment as above -- regarding the capitalization of "GCS" in names.  Thanks.

Reply:
Should be fixed -- I checked both docs when you pointed it out on the first.